File: 1/main.py
# Brandon Blackburn
# Euler Problem #2:
# By considering the terms in the Fibonacci sequence whose values do not exceed four million,
# find the sum of the even-valued terms.

num1 = 1
num2 = 1
answer = 2

while num1 < 4000000 and num2 < 4000000:
    temp = num1 + num2
    answer += temp
    num1 = num2
    num2 = temp

print(answer)

File: 1/Main.java
public class Main {


    public static void main(String args[]){
        int max = 1000;
        int x = 3;
        int y = 5;
        int sum = 0;

        for(int i = 1 ; i < max ; i ++){
            if( (i % 3 == 0) || (i % 5 == 0) )
                sum += i;
        }

        System.out.println(sum);
    }
}
